backend.native/tests/interop/basics/cmacros.def

```diff
@@ -0,0 +1,12 @@
+---
+#define ZERO 0
+#define ONE 1
+#define MAX_LONG 9223372036854775807
+#define FOO_STRING "foo"
```

Comment:
What about wide strings and such, i.e. http://en.cppreference.com/w/cpp/language/string_literal?